src/styles/blocks/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-image {
+  display: block;
+  margin: 16px auto 40px;
+  width: 160px;
+  height: 134px;
```

Comment:
so we stretch image?

Reply:
yes, we do. do we have any other options?
in the next home task probably we can make it depend on card size, but still we will stretch it.